lambda/rotator-snowflake/setup/snowflake/snowflake.go

```diff
@@ -6,65 +6,40 @@ import (
 
 	"github.com/chanzuckerberg/go-misc/errors"
 	"github.com/chanzuckerberg/go-misc/snowflake"
-	"github.com/hashicorp/go-multierror"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/sirupsen/logrus"
 )
 
-func configureConnection(env *SnowflakeClientEnv) (*sql.DB, error) {
-	cfg := snowflake.Config{
-		Account:  env.NAME,
-		User:     env.USER,
-		Role:     env.ROLE,
-		Region:   env.REGION,
-		Password: env.PASSWORD,
+func LoadSnowflakeEnv(acctName string) (*SnowflakeClientEnv, error) {
+	// If acctName has "okta" or "databricks" in the name, print a warning for possible name collision
+	oktaCollision := strings.Contains(acctName, "okta")
+	if oktaCollision {
+		logrus.Warnf("Snowflake Account %s will likely collide with okta Environment Variables", acctName)
 	}
 
+	databricksCollision := strings.Contains(acctName, "databricks")
+	if databricksCollision {
+		logrus.Warnf("Snowflake Account %s will likely collide with databricks Environment Variables", acctName)
+	}
+
+	snowflakeEnv := &SnowflakeClientEnv{}
+
+	err := envconfig.Process(acctName, snowflakeEnv)
```

Comment:
would it be worth forcing a prefix for the accountname of snowflake. something like `envconfig.Process(fmt.Sprintf("snowflake_%s", acctName), snowflakeEnv)`

Reply:
to entirely avoid collisions with DB + Okta